src/cli/new.rs

```diff
@@ -88,7 +93,7 @@ pub mod tests {
         // this test is deterministic
         let dir = gen_dir();
         let dir_path_buf = &dir.path().to_path_buf();
-        setup_test_folder(dir_path_buf, &TEST_DIR_NAME);
+        setup_test_folder(dir_path_buf, &TEST_DIR_NAME).expect("Test folder not set up");
```

Comment:
I think this may cause a panclic, which I'm not sure is what we want

Reply:
Why would it fail to set up the test folder? and what would we want it to do, if it does?

Reply:
I added this only to squelch a warning about an unused Result. So, it's better to panic than nothing at all! This would be really weird if it failed.

Reply:
This is a test. So a panic is okay.

Reply:
Right, haha. oops.